### DISCUSSION TITLE
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object

### DISCUSSION THREAD

**Discussion by hassam-saeed** [vikejs/vike#1410] [Discussion] [answered]
I am working on a project that I am converting from React CSR to SSR using vite-plugin-ssr that I deployed to Vercel. All of the routes are working except a few ones. While fetching these routes I am facing an error like this:

I am fetching the route `/case-studies`

<img width="1672" alt="Screenshot 2024-01-04 at 3 31 00 PM" src="https://github.com/vikejs/vike/assets/51261422/e952440f-e9db-4645-b4f3-68a17969e53d">

Want to take a look at the repo? Repo is here: 
https://github.com/DevAxonOfficial/SSR-Frontend

**Comment by lourot** [comment]
Hi @hassam-saeed, this is usually a "named import vs. default import" problem. See for example:

* https://github.com/vikejs/vike/discussions/901#discussioncomment-8001393
* https://vike.dev/broken-npm-package
* https://github.com/vikejs/vike/pull/1358/files

**Reply by brillout** [reply]
@hassam-saeed Also see https://github.com/vikejs/vike/discussions/901#discussioncomment-8001141 which we'll merge today.

**Reply by hassam-saeed** [reply]
@AurelienLourot Can you please take a look at the ```/case-studies``` route and point out the import issue?

**Reply by lourot** [reply]
Please try yourself first. We have pointed you at links only 15 minutes ago. It seems you have really really tried yourself to wrap your head around the problem

**Reply by hassam-saeed** [reply]
I have found the issue that there is an issue with react-slick package
<img width="720" alt="Screenshot 2024-01-04 at 6 57 31 PM" src="https://github.com/vikejs/vike/assets/51261422/8cc6b9b4-d0c4-4162-91d9-d966bec2f19e">

**Reply by phonzammi** [reply, marked as answer]
```ts
import Slider from 'react-slick';
// ...
function Client_slider() {
    const SliderComponent = typeof window === 'undefined' ? Slider.default : Slider;
// ...
<SliderComponent {...custm_slider} className=' custm_main_slider social_main_slider'>
// ...
</SliderComponent>
}
```
from [react-slick issue](https://github.com/akiran/react-slick/issues/2206)

**Reply by hassam-saeed** [reply]
Thank you @phonzammi. It is working fine for me
